src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -99,63 +99,87 @@ public override ICommandLineParserResult ParseArgs(string[] args)
             return Parser.Parse(args);
         }
 
+        public static IWebHost BuildWebHost(string url, IConfiguration configuration) =>
+            Microsoft.AspNetCore.WebHost.CreateDefaultBuilder(new string[0])
```

Comment:
Currently passing an empty array, but we could allow flowing arguments to the builder.